src/lib/Backpack/devBackpack.cpp

```diff
@@ -36,7 +35,8 @@ void startPassthrough()
 
     // get ready for passthrough
     CRSF::Port.begin(460800, SERIAL_8N1, GPIO_PIN_RCSIGNAL_RX, GPIO_PIN_RCSIGNAL_TX);
-    LoggingBackpack.begin(460800, SERIAL_8N1, GPIO_PIN_DEBUG_RX, GPIO_PIN_DEBUG_TX);
+    HardwareSerial &backpack = *((HardwareSerial*)LoggingBackpack);
+    backpack.begin(460800, SERIAL_8N1, GPIO_PIN_DEBUG_RX, GPIO_PIN_DEBUG_TX);
```

Comment:
@CapnBry got any better suggestions for this?

Reply:
😬 Well, the LoggingBackpack already runs at `#define BACKPACK_LOGGING_BAUD 460800` so you'd just replace the line with:
```
#if BACKPACK_LOGGING_BAUD != 460800
#error "Passthrough backpack requires BACKPACK_LOGGING_BAUD==460800
#endif
```

All the code after that would still operate on `LoggingBackpack->`.